pkgs/by-name/ff/ffpb/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "ffpb";
+  version = "0.4.1";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "7eVqbLpMHS1sBw2vYS4cTtyVdnnknGtEI8190VlXflk=";
+  };
+
+  propagatedBuildInputs = [ python3Packages.tqdm ];
```

```suggestion
  build-system = [ python3Packages.setuptools ];

  dependencies = [ python3Packages.tqdm ];
```
I recently learned that even packages without pyproject.toml can (and should) use the new pyproject builder.